package.json

```diff
@@ -25,7 +24,7 @@
     "fh-cls-mongoose": "2.1.1",
     "fh-gridfs": "1.1.1",
     "fh-logger": "0.5.0",
-    "fh-mbaas-client": "0.15.0",
+    "fh-mbaas-client": "1.1.1",
```

Comment:
Is this bump in fh-mbaas-client needed? I thought it was fh-supercore > fh-mbaas-client > fh-mbaas > fh-forms.

Reply:
Needed? Probably not. I don't see why we wouldn't bump it though, given that I've made changes to fh-mbaas-client for fh-supercore's interaction with it, as part of this same body of work. If we _don't_ bump it, then it just falls further behind, and results in 2 different copies of fh-mbaas-client (& probably some dependencies) below fh-supercore. When they're the same version, they'll be deduplicated.